src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterApplicationBuilderExtensions.cs

```diff
@@ -21,38 +21,133 @@
 using Microsoft.Extensions.DependencyInjection;
 using OpenTelemetry.Exporter;
 using OpenTelemetry.Exporter.Prometheus;
+using OpenTelemetry.Internal;
 using OpenTelemetry.Metrics;
 
 namespace Microsoft.AspNetCore.Builder
 {
     /// <summary>
-    /// Provides extension methods for <see cref="IApplicationBuilder"/> to add Prometheus Scraper Endpoint.
+    /// Provides extension methods for <see cref="IApplicationBuilder"/> to add
+    /// Prometheus scraping endpoint.
     /// </summary>
     public static class PrometheusExporterApplicationBuilderExtensions
     {
         /// <summary>
         /// Adds OpenTelemetry Prometheus scraping endpoint middleware to an
         /// <see cref="IApplicationBuilder"/> instance.
         /// </summary>
+        /// <remarks>Note: A branched pipeline is created for the route
+        /// specified by <see
+        /// cref="PrometheusExporterOptions.ScrapeEndpointPath"/>.</remarks>
+        /// <param name="app">The <see cref="IApplicationBuilder"/> to add
+        /// middleware to.</param>
+        /// <returns>A reference to the original <see
+        /// cref="IApplicationBuilder"/> for chaining calls.</returns>
+        public static IApplicationBuilder UseOpenTelemetryPrometheusScrapingEndpoint(this IApplicationBuilder app)
+            => UseOpenTelemetryPrometheusScrapingEndpoint(app, meterProvider: null, predicate: null, path: null, configureBranchedPipeline: null);
+
+        /// <summary>
+        /// Adds OpenTelemetry Prometheus scraping endpoint middleware to an
+        /// <see cref="IApplicationBuilder"/> instance.
+        /// </summary>
+        /// <remarks>Note: A branched pipeline is created for the supplied
+        /// <paramref name="path"/>.</remarks>
+        /// <param name="app">The <see cref="IApplicationBuilder"/> to add
+        /// middleware to.</param>
+        /// <param name="path">Path to use for the branched pipeline.</param>
```

Comment:
we fallback to `ScrapeEndpointPath`, if path is not given, right?

Reply:
* In the case of the extension which accepts path, you are required to specify it:

```csharp
public static IApplicationBuilder UseOpenTelemetryPrometheusScrapingEndpoint(this IApplicationBuilder app, string path)
{
   Guard.ThrowIfNull(path);
```

I figured if user was calling the path extension, without path, they probably made a mistake!

* In the case of the extension where you can specify everything, there is fallback logic:

```csharp
            if (predicate == null) // Respect predicate if set
            {
                if (path == null) // Respect path if set
                {
                    var options = app.ApplicationServices.GetOptions<PrometheusExporterOptions>();

                    // Respect options if set otherwise fallback to default
                    path = options.ScrapeEndpointPath ?? PrometheusExporterOptions.DefaultScrapeEndpointPath;
                }
```